doc.ddoc

```diff
@@ -71,6 +71,7 @@ $(DIVID cssmenu, $(UL
         dmd-windows.html, dmd $(NDASH) reference compiler,
         http://gdcproject.org, gdc $(NDASH) gcc-based compiler,
         http://wiki.dlang.org/LDC, ldc $(NDASH) LLVM-based compiler,
+        http://code.dlang.org, dub $(NDASH) D package manager, 
```

Comment:
We already have a code.dlang.org link above, though "more libraries" should be changed to "DUB - The D package registry".
You might put a link to http://code.dlang.org/download in the tool section.